react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js

```diff
@@ -102,7 +102,14 @@ export default class JitsiStreamBackgroundEffect {
         } else {
             this._outputCanvasCtx.filter = 'blur(8px)';
         }
+        if (this._options.virtualBackground.backgroundType === VIRTUAL_BACKGROUND_TYPE.DESKTOP_SHARE) {
```

Comment:
Since `this._options.virtualBackground.backgroundType` is used multiple times in` runPostProcessing()`, can you use just the `backgroundType` by destructuring in the beginning ?
`const { backgroundType } = this._options.virtualBackground;`